Library/QuickPulseSender.ts

```diff
@@ -84,6 +88,19 @@ class QuickPulseSender {
             additionalHeaders.forEach(header => options.headers[header.name] = header.value);
         }
 
+        if (this._authorizationHandler) {
+            try {
+                // Add bearer token
+                await this._authorizationHandler.addAuthorizationHeader(options);
+            }
+            catch (authError) {
+                let notice = `Failed to get AAD bearer token for the Application. Error:`;
+                Logging.info(QuickPulseSender.TAG, notice, authError);
```

Comment:
Shouldnt this be Logging.warn?  I think its critical enough to be a warning ...there will be data loss.

Reply:
This is for quickpulse calls, will apply to both ping and post requests so warnings could get super noisy, data loss is not critical here